rickshaw-settings.json

```diff
@@ -20,7 +20,11 @@
 	}
     },
     "workshop": {
-      "force-builds": "false"
+        "force-builds": "false",
+        "refresh-timestamps": {
+            "token-file": null,
+            "api-url": null
+        }
     },
     "quay": {
         "image-expiration": "2w"
```

Comment:
2w for unused images ? if no runs, it expires in 2w, right ?

Reply:
Yes, unless it was built elsewhere (such as CI) with a longer expiration.